src/js/search.js

```diff
@@ -1,14 +1,11 @@
-import { API } from './api';
-import renderFilmsMarkup from './renderMovieCards';
-// import renderFilmsMarkup from ''; TO DO: (add genres?)
+import Api from './api.js';
+import renderFilmsMarkup from './search-features/renderFilmsMarkup.js';
 
 const searchFormRef = document.querySelector('.form');
 const errorMessage = document.querySelector('.form__search-error');
-const filmsApi = new API();
+const filmsApi = new Api();
```

Comment:
Nie tworzymy nowej instancji z nazwą filmsApi, jest to zbędne i tak całe Api jest związane z filmami. Ten moduł działa tak po imporcie, który jest tutaj w porządku piszemy Api.{ tutaj nazwa funkcji } -> np. `Api.getTrendingMovies`